src/server/utilities/cspHeader/index.js

```diff
@@ -86,13 +93,20 @@ const directives = {
       'https://emp.bbc.co.uk',
       'https://chartbeat.com',
       'https://*.chartbeat.com',
+      'https://www.youtube.com', // Social Embeds
+      'https://platform.twitter.com', // Social Embeds
+      'https://www.instagram.com', // Social Embeds
+      'https://syndication.twitter.com', // Social Embeds
       "'self'",
     ],
     ampNonLive: [
       'https://polling.bbc.co.uk', // Media page
       'https://polling.test.bbc.co.uk', // Media page
       'https://securepubads.g.doubleclick.net', // ads
       'https://tpc.googlesyndication.com', // ads
+      'https://www.youtube.com', // Social Embeds, <amp-youtube />
+      'https://www.instagram.com', // Social Embeds, <amp-instagram />
+      'https://*.ampproject.net', // Social Embeds
```

Comment:
This array is for the Canonical page on Live, so you don't need this amp domain here
```suggestion
```